### DISCUSSION TITLE
--restart run with a new learning rate?

### DISCUSSION THREAD

**Discussion by Lewis-YL** [deepmodeling/deepmd-kit#1703] [Q&A] [answered]
Hi guys,

Is there a way to reset the index of the checkpoint when using --restart?

According to the manual,
lr(t) = start_lr * decay_rate ^ ( t / decay_steps )

When t is 1 million steps or even larger, 
it is hard to adjust the rate at the checkpoint based on start_lr and decay_steps.

**Comment by Lewis-YL** [comment, marked as answer]
Problem solved.

**Reply by lnnbig** [reply]
Hi, may I ask how to restart with a new learning rate? Many thanks!
